cdap-ui/app/cdap/components/Header/ProductDropdown/index.js

```diff
@@ -143,13 +141,7 @@ export default class ProductDropdown extends Component {
           toggle={this.toggleCdapMenuDropdown.bind(this)}>
           <DropdownToggle caret>
             <div className="secure-mode-icon">
-              <IconSVG name={ window.CDAP_CONFIG.securityEnabled ? "icon-lock_close" : "" } />
-              <div className="cdap-mode">{mode}</div>
-            </div>
-            <div className="cdap-logo-container">
-              <div className="caret-down-container">
-                <IconSVG name="icon-caret-down" />
-              </div>
+              <IconSVG name="icon-cog" />
```

Comment:
we're using the multiple cog wheel right?

Reply:
I see. ok changing it